bitwarden_license/src/Commercial.Core/SecretManagerFeatures/AccessTokens/CreateAccessTokenCommand.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Bit.Core.Entities;
+using Bit.Core.Repositories;
+using Bit.Core.SecretManagerFeatures.AccessTokens.Interfaces;
+using Bit.Core.Utilities;
+
+namespace Bit.Commercial.Core.SecretManagerFeatures.AccessTokens;
+
+public class CreateAccessTokenCommand : ICreateAccessTokenCommand
+{
+    private readonly int _dbMaxLength = 30;
```

Comment:
It's not clear that this is the client secret max length. `_clientSecretLength` perhaps?

Reply:
Resolved in https://github.com/bitwarden/server/pull/2411/commits/de568039e7031da247862455f059aed23fa3dc0d